### ISSUE TITLE
Remove signer log when DKG doesn't succeed

### ISSUE THREAD

**Issue by hstove** [stacks-network/stacks-core#4757]
When a new 2.5 reward cycle starts and the aggregate key isn't set, the signer has this log every few seconds:
 
 ```
Signer #N: Could not lookup current miner while in active reward cycle
```

I thought we'd removed this log during 2.5, but looks like we haven't.

Relates to:

- #4729

**Comment by wileyj** [comment]
#4760 

@hstove is it feasible to have a PR addressing this for next week?

**Comment by diwakergupta** [comment]
Fixed by #4768
